### PR TITLE
Update HTML file and CSS file

### PR DESCRIPTION
**Pull request by ShiroYaksha90**
Make work, aboutme, form sections responsive to desktop Modify 'bottom' part inside aboutme section

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ShiroYaksha90 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptivr PR `title` and `summary`
- [ ] Good looking UI :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -1138,17 +770,669 @@ height: 48px; */
...
+  }
+
+  .box2 {
+    font-weight: 500;
+    font-size: 32px;
+    width: 466px;
+    height: 24px;
+    margin: 0 46px 0 0;
+    line-height: 44px;
+  }
```
Comment (emmiiorji) [inline styles.css:863]: - [ ] You have made a great effort to adjust the margin of the heading text as requested by the previous reviewer. However, it's yet to match the design. Consider reducing the top margin of the `destop-constants` class and the top margin of the `box1` class.

- [ ] The horizontal scroll the appears on the page. Kindly adjust this as requested by the previous reviewer in order to offer users better experience.

Your design:
image

Figma design:
'

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @ShiroYaksha90, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

a little more

 ## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @ShiroYaksha90, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**